src/dust/dust_cuda.cu

```diff
@@ -1,20 +1,30 @@
+/*!
+ * \file dust_cuda.cu
+ * \author Helena Richie (helenarichie@gmail.com)
+ * \brief Contains code that updates the dust density scalar field. The dust_kernel function determines the rate of
+ * change of dust density, which is controlled by the sputtering timescale. The sputtering timescale is from the
+ * McKinnon et al. (2017) model of dust sputtering, which depends on the cell's gas density and temperature.
+ */
+
 #ifdef CUDA
```

Comment:
I am actually trying to deprecate the CUDA macro (see Issue #241) since in practice Cholla can only be run in GPU mode these days, so let's go ahead and get rid of it in new code.